scripts/foundry/governance/RevokeMultiSig.s.sol

```diff
@@ -0,0 +1,266 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.19;
+
+import { console } from "forge-std/console.sol";
+import { IVaultManagerFunctions } from "borrow/interfaces/IVaultManager.sol";
+import { IAgToken } from "borrow/interfaces/IAgToken.sol";
+import { IERC721Metadata } from "oz/token/ERC721/extensions/IERC721Metadata.sol";
+import { IAccessControl } from "oz/access/IAccessControl.sol";
+import { Enum } from "safe/Safe.sol";
+import { MultiSend, Utils } from "../Utils.s.sol";
+import "../Constants.s.sol";
+
+interface ITreasuryWithRole {
+    function addMinter(address minter) external;
+
+    function removeMinter(address minter) external;
+}
+
+contract RevokeMultiSig is Utils {
+    address public gaugeSushiAngleAgEUR = 0xBa625B318483516F7483DD2c4706aC92d44dBB2B;
+
+    function run() external {
+        bytes memory transactions;
+        uint8 isDelegateCall = 0;
+        uint256 value = 0;
+        address to;
+
+        /** TODO  complete */
+        uint256 chainId = CHAIN_ETHEREUM;
+        /** END  complete */
+
+        CoreBorrow core = CoreBorrow(_chainToContract(chainId, ContractType.CoreBorrow));
+        bytes32 governorRole = core.GOVERNOR_ROLE();
+        bytes32 guardianRole = core.GUARDIAN_ROLE();
+
+        /** Add minting agEUR privilege to the on chain governance */
+        {
+            to = _chainToContract(chainId, ContractType.TreasuryAgEUR);
```

Comment:
We'll need to be wary because I don't think that the governance msig has the minting right on the agEUR contract on every chain.
Only on chains with AMOs